toolkit/compare_case_groups/build_dataframes.py

```diff
@@ -133,14 +118,13 @@ def build_attribute_df(
     )
 
 
-def filter_df(main_df, filters: list[str]):
-    filtered_df = main_df.copy()
-
+def filter_df(main_df: pl.DataFrame, filters: list[str]) -> pl.DataFrame:
     if len(filters) == 0:
```

Comment:
can be replaced by `if filters:`.

In python empty sequences (len == 0) is False

Reply:
Actually this if filter can be removed since it will not go into the loop anyway right

Reply:
you are right :)